docs/getting-started/advanced-setting.asciidoc

```diff
@@ -168,3 +174,9 @@ If you've ensured that your detection rules have the required privileges across
 == Show/hide related integrations in Rules page tables
 
 By default, Elastic prebuilt rules in the *Rules* and *Rule Monitoring* tables include a badge showing how many related integrations have been installed. Turn off `securitySolution:showRelatedIntegrations` to hide this in the rules tables (related integrations will still appear on rule details pages).
+
+[discrete]
+[[manage-alert-tags]]
+== Manage alert tag options
+
+The `securitySolution:alertTags` field determines which options display in the alert tag menu. The default alert tag options are **Duplicate**, **False positive**, and **Further investigation required**. You can update the alert tag menu by editing these options or adding more.
```

Comment:
In 8.9.0 BC4, the casing is slightly different:

<img width="600" alt="Screenshot 2023-07-13 at 2 24 26 PM" src="https://github.com/elastic/security-docs/assets/79124755/ab549e7d-ebf2-4c8b-9845-26aa16da490f">

Will need to change this to:

```suggestion
The `securitySolution:alertTags` field determines which options display in the alert tag menu. The default alert tag options are **Duplicate**, **False Positive**, and **Further investigation required**. You can update the alert tag menu by editing these options or adding more.
```